task.sh

```diff
@@ -1 +1,42 @@
 #! /bin/bash
+
+BACKUP_PATH="ShopDB_backup.sql"
+DATA_BACKUP_PATH="ShopDB_data_backup.sql"
+
+if [ -z "$DB_USER" ] || [ -z "$DB_PASSWORD" ]; then
+  echo "Error: Environment DB_USER or DB_PASSWORD do not exist."
+  exit 1
+fi
+
+check_error() {
+  if [ $? -ne 0 ]; then
+    echo "Error: $1"
+    exit 1
+  fi
+}
+
+echo "Creating full backup of ShopDB..."
+mysqldump -u "$DB_USER" -p"$DB_PASSWORD" \
```

Comment:
It's a good practice to quote the database names and file paths in case they contain spaces or special characters. Although it's unlikely for database names, it's a good habit for file paths.